spec/features/users/show_spec.rb

```diff
@@ -1,22 +1,106 @@
-require "rails_helper"
+require 'rails_helper'
 
-RSpec.describe "User dashboard" do
+RSpec.describe 'User dashboard' do
   before do
-    @ally = User.create!(name: "Ally Jean", email: "allyjean@example.com")
+    @ally = User.create!(name: 'Ally Jean', email: 'allyjean@example.com')
     visit user_path(@ally)
   end
 
-  it "has a heading" do
+  it 'has a heading' do
     expect(page).to have_content("Ally Jean's Dashboard")
   end
-  
-  it "has a button to discover movies" do
-    expect(page).to have_button("Discover Movies")
+
+  it 'has a button to discover movies' do
+    expect(page).to have_button('Discover Movies')
+  end
+
+  it 'has a section that lists viewing parties' do
```

Comment:
You can delete this test scaffold since you made your own instead

Reply:
Done